src/storage/mod.rs

```diff
@@ -1566,17 +1604,16 @@ mod tests {
                 .async_get(Context::new(), Key::from_raw(b"x"), 101)
                 .wait(),
         );
-        storage.stop().unwrap();
```

Comment:
Will it be properly stopped if we remove this line? I didn't understand clearly 🤔

Reply:
It won't be properly stopped. However we will handle it in #3756

Reply:
How about remove these stops when it when it was done, but not in this PR?

Reply:
I think both are ok, because there are already a lot of tests not closing it 🤣